monai/config/print_dependencies.py

```
@@ -0,0 +1,64 @@
+
+import sys
+
+BUILD_SYSTEM_KEY = "build-system"
+PROJ_KEY = "project"
+OPTS_KEY = "optional-dependencies"
+DEP_KEY = "dependencies"
+REQ_KEY = "requires"
+TOML_FILE = "pyproject.toml"
+
+
+def parse_dependencies(filename=TOML_FILE, sections=[]):
```

Ruff (0.15.21)

[warning] Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)